docs/wiki/user/radio-signal-loss.md

```diff
@@ -10,6 +10,11 @@ Every radio also has its own transmission power which determines the initial str
 
 The antenna direction for infantry is simulated as being relative to the direction of their back. If radio antenna direction is not ignored (see below) this can be observed by having difficulties receiving transmissions from other players who are prone if you are standing and vice versa as the antennas will be out of alignment.
 
+(Only in Dev-Build): However it is possible to manually align the antenna by bending it in order to improve signal quality. To keep it as simple as possible, there are only two possible antenna alignments, the player can choose between the two via ACE Interaction Menu or Custom Keybind (Default: `CTRL+ALT+UP`). In addition to the manual method, there is always the possibility to activate an automatic antenna alignment via CBA Settings (see below). The current alignment of the antenna is shown in the Arma 3 Stance Indicator. 
```

Comment:
```suggestion
(Only in Dev-Build): However it is possible to manually align the antenna by bending it in order to improve signal quality. To keep it as simple as possible, there are only two possible antenna alignments, the player can choose between the two via ACE Interaction Menu or Custom Keybind (default: `CTRL+ALT+UP`). In addition to the manual method, there is always the possibility to activate an automatic antenna alignment via CBA Settings (see below). The current alignment of the antenna is shown in the Arma 3 Stance Indicator. 
```

Reply:
Can't detect any difference in the suggestion?

Reply:
Ah, ok. Trailing whitespace